### PR TITLE
Add  upload and download and other command's logs

### PR DESCRIPTION
**Pull request by gongweibao**

### CODE REVIEW

File: go/filemanager/pfsserver/handler.go [hunk 1]
```diff
@@ -341,6 +367,8 @@ func GetChunkHandler(w http.ResponseWriter, r *http.Request) {
 	}
 	defer fr.Close()
 
+	log.Infof("user:%s download %s\n", user, p.String())
```
Comment (typhoonzero) [inline go/filemanager/pfsserver/handler.go]: Curious how did you configure to output logs to cephfs files?

File: go/filemanager/pfsmodules/cp.go [hunk 2]
```diff
@@ -45,6 +46,22 @@ func newCpCmdFromFlag(f *flag.FlagSet) (*CpCmd, error) {
...
+			if err != nil {
+				log.Errorln("meets error when parsing argument v")
+				return
+			}
+
+			cmd.ChunkSize = cmd.ChunkSize * 1024
+
+			if cmd.ChunkSize > defaultMaxChunkSize ||
+				cmd.ChunkSize < defaultMinChunkSize {
+				log.Errorln("ChunkSize should be in [%d, %d]", defaultMinChunkSize, defaultMaxChunkSize)
```
[the remark this answers is not in the document]
Comment (gongweibao) [reply go/filemanager/pfsmodules/cp.go]: 确实。Done。

File: go/filemanager/pfsserver/handler.go (hunk 1 above)
Comment (gongweibao) [reply go/filemanager/pfsserver/handler.go]:
```
      - name: logs-storage
        cephfs:
          monitors: ["xx.xx.xx.xx:6789"]
          user: "admin"
          path: "/gongweibao@baidu.com/pfsserver/logs"
          secretRef:
            name: ceph-secret
      containers:
      - name: pfs
        imagePullPolicy: Always
        image: bootstrapper:5000/paddlepaddle/pfsserver-test:latest
        env:
          - name: CURRENT_DATACENTER
            value: "dlnel"
        #command: ["/pfsserver/pfsserver", "--port", "8083", "-tokenuri", "http://paddle-cloud-service:8000", "-logtostderr=true", "-log_dir=./log", "-v=4"]
        command: ["/pfsserver/pfsserver", "--port", "8083", "-tokenuri", "http://paddle-cloud-service:8000", "-log_dir=/logs", "-alsologtostderr=true"]
        volumeMounts:
        - name: data-storage
          mountPath: /pfs/dlnel/home
        - name: public-data-storage
          mountPath: /pfs/dlnel/public/dataset
        - name: logs-storage
          mountPath: /logs
```

Comment (gongweibao) [reply go/filemanager/pfsserver/handler.go]: 日志收集需要有一个方案出来。https://github.com/PaddlePaddle/cloud/issues/357

**Pull request merged**